hackvote/db/migrate/20150521210620_create_events.rb

```diff
@@ -0,0 +1,9 @@
+class CreateEvents < ActiveRecord::Migration
+  def change
+    create_table :events do |t|
+      t.string :title
```

Comment:
Hmm, so append null: false to the columns of the table? Does this restrict that null values can't be stored in this column?